harmony/light.css

```diff
@@ -0,0 +1,207 @@
+[data-theme="light"] {
```

Comment:
It should be possible to avoid duplication of whole theme in this file. Why `[data-theme="light"]` is needed anyway?